exercises/high-scores/tests/high-scores.rs

```diff
@@ -11,14 +11,21 @@ fn test_list_of_scores() {
 #[ignore]
 fn test_latest_score() {
     let high_scores = HighScores::new(&[100, 0, 90, 30]);
-    assert_eq!(high_scores.latest(), 30);
+    assert_eq!(high_scores.latest(), Some(30));
 }
 
 #[test]
 #[ignore]
-fn test_personal_best() {
```

Comment:
I am surprised to see that you removed `test_personal_best` entirely. While I entirely approve of the addition of `test_latest_score_empty`, `test_personal_best_empty`, and `test_personal_top_three_empty`, I think we should not remove this canonical test case.

Reply:
I'm pretty sure it was by accident. Thanks a lot!